frontend/src/layout/navigation-3000/sidepanel/SidePanel.tsx

```diff
@@ -70,6 +70,13 @@ export function SidePanel(): JSX.Element | null {
 
     const { desiredWidth, isResizeInProgress } = useValues(resizerLogic(resizerLogicProps))
 
+    useEffect(() => {
+        setSidePanelAvailable(true)
```

Comment:
The logic is mounted all over the place, so here we can make it super clear that the component itself is available